src/K9OCRS/Controllers/ClassSectionsController.cs

```diff
@@ -52,8 +52,8 @@ public async Task<IActionResult> GetAll()
                 // Get all sections
                 var sections = await dbOwner.ClassSections.GetAll(conn);
                 // Get all instructors associated to sections
-                //var instructorIds = sections.Select(s => s.InstructorID);
-                //var instructors = (await dbOwner.Users.GetByIDs(conn, instructorIds)).ToList();
+                var instructors = (await dbOwner.Users.GetByIDs(conn, instructorId)).ToList();
+                var instructorIds = sections.Select(s => s.InstructorID);
```

Comment:
This change should be reverted, I haven't finished this endpoint yet so uncommenting these lines is causing a build error.
```suggestion
                //var instructorIds = sections.Select(s => s.InstructorID);
                //var instructors = (await dbOwner.Users.GetByIDs(conn, instructorIds)).ToList();
```